### PR TITLE
fix(highlights): SD-4535 Showing only highlight type of templates

### PR DESCRIPTION
**Pull request by pavlovicnemanja**
SD-4535

### CODE REVIEW

**Comment by akintolga** [comment]
:+1:

**Comment by sjunaid** [comment]
:+1:

File: scripts/superdesk-ingest/views/dashboard/ingest-dashboard-widget.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 <div ng-show="item.show_status" class="status" ng-class="{closed: item.is_closed, opened: !item.is_closed}">
     <div ng-show="item.is_closed">
-        <div><span class="uppercase" translate>Closed</span><span> by {{item.last_closed.display_name || 'System'}}</span></div>
+        <div><span class="uppercase" translate>Closed</span><span>by {{item.last_closed.display_name || 'System'}}</span></div>
```
Comment (mdhaman) [inline scripts/superdesk-ingest/views/dashboard/ingest-dashboard-widget.html]: missing translate.

**Comment by mdhaman** [comment]
:+1:

**Comment by Aca-jov** [comment]
:+1:

**Pull request merged**